models/userModel.js

```diff
@@ -87,15 +87,7 @@ const User = sequelize.define('user', {
   role: {
     type: DataTypes.STRING(255),
     allowNull: false,
-    validate: {
-      notEmpty: {
-        msg: 'Role cannot be empty',
-      },
-      isIn: {
-        args: [['admin', 'customer']],
-        msg: 'Invalid role. Must be either "admin" or "customer".',
-      },
-    },
+    defaultValue: 'admin',
```

Comment:
make the default customer just to be in the safe zone if an error appears when registering a new user.